### DISCUSSION TITLE
Migrate from official self hosted is possible?

### DISCUSSION THREAD

**Discussion by riccardospeggiorin-centropaghe** [dani-garcia/vaultwarden#1384] [Ideas]
Hi, is it possible to migrate from the docker official self hosted installation ?
Thanks

**Comment by tanerpaca** [comment]
I’ve done it very recently without any problem, just export vault and import in bitwarden_rs, took me few seconds.

**Comment by riccardospeggiorin-centropaghe** [comment]
mm ok but I am not asking for my own vault, but for all the users's vaults

**Reply by BlackDex** [reply]
Then all the users need to export > import it them selfs. Also owners or admin's of organizations need to export those vaults also.
You need to recreate the users and orgs and invite all the users again. There is no easy way to transfer all the data.
